modules/userId/index.js

```diff
@@ -605,6 +605,57 @@ function getUserIdsAsEids() {
   return createEidsArray(getCombinedSubmoduleIds(initializedSubmodules));
 }
 
+/**
+ * This function will be exposed in global-name-space so that userIds for a source can be exposed
+ * Sample use case is exposing this function to ESP
+ */
+function getEncryptedEidsForSource(source, encrypt, customFunction) {
+  let eids = [];
+  let eidsSignals = {};
+
+  if (typeof customFunction === 'function') {
+    logInfo(`${MODULE_NAME} - Getting encrypted signal from custom function : ` + customFunction.name + `& source : ` + source);
```

Comment:
We can avoid concatenating using + by using Template literal.

Reply:
Done